README.md

```diff
@@ -122,7 +129,7 @@ Common Scenarios
 
 ## All done? Connect a device!
 By default, the solution uses simulated devices. You can start adding your
-own devices with the instructions here: [Connect a physical device](https://docs.microsoft.com/en-us/azure/iot-suite/iot-suite-connecting-devices-linux)
+own devices with the instructions here: [Connect a physical device](https://docs.microsoft.com/azure/iot-suite/iot-suite-connecting-devices-linux)
```

Comment:
The links in the README are listed at the bottom of the markdown file. Please add the link to the bottom of the page with an identifier, for example:
`[build-badge]: https://img.shields.io/travis/Azure/azure-iot-pcs-remote-monitoring-dotnet.svg`